src/liquid/components/ld-pagination/ld-pagination.shadow.css

```diff
@@ -129,3 +162,65 @@
   );
   transition: transform 100ms ease;
 }
+
+.ld-pagination--dots {
+  --ld-pagination-item-size: var(--ld-pagination-dots-size);
+  --ld-pagination-items-space: var(--ld-pagination-dots-space);
+
+  .ld-pagination__arrow:not(:last-child),
+  .ld-pagination__sticky {
+    margin-right: var(--ld-sp-4);
+  }
+
+  .ld-pagination__slide-wrapper {
+    height: var(--ld-pagination-column-size);
+    margin-right: var(--ld-sp-4);
+  }
+
+  .ld-pagination__item {
+    align-items: center;
+    display: flex;
+    height: var(--ld-pagination-column-size);
+    justify-content: center;
+    left: calc(
+      var(--ld-pagination-item-pos) * var(--ld-pagination-column-size)
+    );
+    width: var(--ld-pagination-column-size);
+
+    &:not(.ld-pagination__item--selected) .ld-pagination__dot::after {
+      background-color: var(--ld-pagination-primary-color);
+      opacity: 0.7;
+    }
+  }
+}
+
+.ld-pagination__dot {
+  align-items: center;
+  display: flex;
+  justify-content: center;
+  height: var(--ld-pagination-column-size);
+  overflow: hidden;
+  width: var(--ld-pagination-column-size);
+
+  &::part(button) {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: var(--ld-pagination-item-size);
+    width: var(--ld-pagination-item-size);
+
+    &:hover {
+      background: transparent;
+    }
```

Comment:
```suggestion
    &:hover {
      background: transparent;
    }

    &:focus {
      outline: auto;
      outline-offset: calc(-1 * var(--ld-sp-12));
      
      &:not(:focus-visible) {
        outline: none;
      }
    }
```

Reply:
Had to do it a little different. Please take a look at the new implementation and see, if you like it.